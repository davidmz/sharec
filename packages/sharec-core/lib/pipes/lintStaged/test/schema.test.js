const { fixtures } = require('testUtils')
const { lintStagedJson } = require('../schema')

describe('pipes > lintStaged > schema', () => {
  describe('JSON', () => {
    const lintStagedBaseFxt = fixtures('lintStaged/json/00-base', 'map')

    it('should merge configs', () => {
      expect(lintStagedJson(lintStagedBaseFxt)).toMatchSnapshot()
    })
  })
})
