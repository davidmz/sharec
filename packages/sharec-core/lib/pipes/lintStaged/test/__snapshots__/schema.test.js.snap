// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pipes > lintStaged > schema JSON should merge configs 1`] = `
Map {
  "src/**/*.{js,jsx,json}" => Array [
    "eslint --fix",
    "prettier --write",
    "git add",
  ],
  "src/**/*.css" => Array [
    "stylelint --fix",
    "prettier --write",
    "git add",
  ],
}
`;
